aws/GitOps/clusters/kubeflow-shared-test-infra-poc-tekton/namespaces/tekton-templates/tf-operator-repo.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
In latest Tekton API, they removed `PipelineResource` object type, so can we remove this file instead?